reports-scheduler/build.gradle

```diff
@@ -164,7 +164,11 @@ dependencies {
     compileOnly "${group}:opensearch-job-scheduler-spi:${job_scheduler_version}"
     implementation group: 'com.google.guava', name: 'guava', version: '31.0.1-jre'
     implementation "org.json:json:20180813"
-    implementation group: 'com.github.wnameless', name: 'json-flattener', version: '0.1.0'
+    implementation group: 'com.github.wnameless.json', name: 'json-flattener', version: '0.15.1'
+    // json-base, jackson-databind, jackson-annotations are transitive dependencies by json-flattener
+    implementation group: 'com.github.wnameless.json', name: 'json-base', version: '2.2.1'
+    implementation "com.fasterxml.jackson.core:jackson-databind:2.13.4"
```

Comment:
Can we use `com.fasterxml.jackson.core:jackson-databind:2.13.4.1` to fix [CVE-2022-42003](https://www.mend.io/vulnerability-database/CVE-2022-42003)?

Reply:
updated to 2.14.1